xwiki-platform-core/xwiki-platform-tree/xwiki-platform-tree-war/src/main/webapp/resources/uicomponents/widgets/tree.js

```diff
@@ -38,6 +38,16 @@ require([paths.treeRequireConfig], function() {
         // Open the tree to the specified node and select it.
         openToNodeId && data.instance.openTo(openToNodeId);
       });
+
+      $('.xtree.dbTreeList').on('changed.jstree', function(event, data) {
```

Comment:
Why limit this to ``.dbTreeList``? I think it's useful anywhere you want to save the selected nodes in a hidden input.

Reply:
Done, simplified the selector.